uktrade/data-hub-frontend
# Feature/cls2 810 remove optional text from mandatory fields in active stage

## Description of change

The 'Specific Investment Programme' is now a mandatory field when moving an investment project from the the 'Active' stage to the "Verify win' stage. This change updates the name displayed for the field so that the text '(optional)' is removed when a project moves to the 'Active' stage to make it clearer that the field is now mandatory.

## Test instructions
1. Select an investment project in the 'Prospect' or 'Assign PM' stage.
Select 'Edit Summary'
At the bottom of the form, the final drop-down field should be named 'Specific investment programme (optional)'
![Screenshot 2024-06-07 at 09 42 15](https://github.com/uktrade/data-hub-frontend/assets/74198488/098d75ae-1a38-4a0c-a6e6-07cd091538e5)

2. Select an investment project in the 'Active', 'Verify win' or 'Won' stage
Select 'Edit Summary'
At the bottom of the form, the final drop-down field should be named 'Specific investment programme'
![Screenshot 2024-06-07 at 09 47 57](https://github.com/uktrade/data-hub-frontend/assets/74198488/dbc92a9b-617f-46d0-9601-aba051c383f2)


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
